sample/Readme.md

```diff
@@ -17,7 +17,7 @@ To give a quick try to this sample, run the following set of commands (assuming
 
 ```bash
 docker-compose --profile oracle up -d
-docker compose exec bitcoind /scripts/create_wallets.sh
+docker-compose exec bitcoind /scripts/create_wallets.sh
```

Comment:
I think `docker compose` is the correct syntax for new docker versions. IIRC `docker-compose` was for older version. So maybe it would be better to fix the line above.

Reply:
I'm not sure which one is proper, but on my default docker install on recent Ubuntu, there is no `compose` command.
`Docker version 24.0.7, build 24.0.7-0ubuntu4.1`
Maybe `docker compose` with a note that `docker-compose` is an (older) alternative?

Reply:
Looking into it a bit more, it seems like `docker-compose` was for compose v1 which is apparently deprecated. I'm fine with having a note for it though it you find it helpful.
https://docs.docker.com/compose/releases/migrate/

Reply:
Note that `docker-compose` is used and mentioned in 2 script files and 2 doc files as well, I'm not touching those here.
This PR updated as suggested,